plugins/org.jkiss.dbeaver.ui.editors.data/OSGI-INF/l10n/bundle.properties

```
@@ -243,4 +245,6 @@ confirm.reset_panels_content.title = Proceed and reset panel contents
 confirm.reset_panels_content.tip = Warns about panels with unsaved changes closing
 
 confirm.close_resultset_edit.message = Result set was edited. Do you want to save changes in the database?
-confirm.close_resultset_edit.title = Save result set edit
\ No newline at end of file
+confirm.close_resultset_edit.title = Save result set edit
+
+org.jkiss.dbeaver.toolBarConfiguration.resultsetViewer.bottom = Result set bottom toolbar
```

Missing trailing line

```suggestion
confirm.close_resultset_edit.title = Save result set edit

org.jkiss.dbeaver.toolBarConfiguration.resultsetViewer.bottom = Result set bottom toolbar

```